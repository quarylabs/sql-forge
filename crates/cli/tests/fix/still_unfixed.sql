SELECT
    sum(b)
FROM
    tests;
