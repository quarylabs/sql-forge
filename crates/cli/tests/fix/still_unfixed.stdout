1 files processed, nothing to fix.
